configuration/analyzer_config.json

```diff
@@ -893,6 +893,49 @@
       }
     }
   },
+  "DocGuard_Get": {
+    "type": "observable",
+    "python_module": "docguard_get.DocGuard_Hash",
+    "description": "check if an hash was analyzed on DocGuard. [DocGuard](https://www.docguard.io)",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": false,
+    "observable_supported": [
+      "hash"
+    ],
+    "config": {
+      "soft_time_limit": 30,
+      "queue": "default"
+    },
+    "secrets": {
+      "api_key_name": {
+        "env_var_key": "DOCGUARD_KEY",
+        "description": "",
+        "required": false
+      }
+    },
+    "params": {}
+  },
+  "DocGuard_Upload_File": {
+    "type": "file",
+    "python_module": "docguard.DocGuardUpload",
+    "description": "Analyze files in seconds. [DocGuard](https://www.docguard.io) Please register and use api-key to your privacy",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": false,
+    "config": {
```

Comment:
can you please provide us the mimetypes accepted by your service? This should be added here in the `supported_filetypes` parameter, as stated in the docs.

Otherwise, for instance, the user would also upload executable files to your service and this would not make any sense.